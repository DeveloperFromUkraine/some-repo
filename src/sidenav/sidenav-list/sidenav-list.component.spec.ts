import { ComponentFixture, TestBed } from '@angular/core/testing';
import { SideNavListComponent } from './index';
import { ComponentTest } from '../../../test/test-bed/component';

describe('Side Nav List', () => {
  let fixture: ComponentFixture<SideNavListComponent>;

  beforeEach(async () => {
    await ComponentTest.createTestBed([], [SideNavListComponent]);
  });

  beforeEach(() => {
    fixture = TestBed.createComponent(SideNavListComponent);
  });

  it('should match snapshot', () => {
    expect(fixture).toMatchSnapshot();
  });
});
