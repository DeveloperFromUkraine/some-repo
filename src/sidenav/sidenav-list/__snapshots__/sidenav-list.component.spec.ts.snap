// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Side Nav List should match snapshot 1`] = `
<ign-sidenav-list>
  
</ign-sidenav-list>
`;
